source/fundamentals/connections/tls.txt

```diff
@@ -0,0 +1,199 @@
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect to MongoDB instances with
+the TLS security protocol.
+
+To configure your connection to use TLS, enable
+the TLS option and provide your certificates for validation.
+
+.. tip::
+   
+   To learn more about TLS, see the Wikipedia entry on
+   :wikipedia:`Transport Layer Security <w/index.php?title=Transport_Layer_Security&oldid=1184063676>`.
+
+.. _golang-enable-tls:
+
+Enable TLS
+----------
+
+You can enable TLS on a connection to your MongoDB instance
+in the following ways:
+
+- Setting the ``tls`` option to ``true`` in your connection string
+- Passing an empty ``Config`` struct to the ``SetTLSConfig()``
+  method when creating a ``ClientOptions`` instance
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connection string tls true
+
+      A ``Client`` instance can connect with TLS if you set the
+      ``tls`` option to ``true`` in your connection string:
+
+      .. code-block:: go
+         :emphasize-lines: 1
+
+         uri := "mongodb://<hostname>:<port>?tls=true"
+         opts := options.Client().ApplyURI(uri)
+         client, _ := mongo.Connect(context.TODO(), opts)
+
+   .. tab:: ClientOptions
+      :tabid: clientoptions tls true
+
+      A ``Client`` instance can connect with TLS if you set the
+      ``TLSConfig`` field of a ``ClientOptions`` instance to an empty
+      ``Config`` struct:
+      
+      .. code-block:: go
+         :emphasize-lines: 2
+         
+         uri := "<connection string>"
+         opts := options.Client().ApplyURI(uri).SetTLSConfig(&tls.Config{})
+         client, _ := mongo.Connect(context.TODO(), opts)
+
+.. note::
+   
+   If you use a DNS SRV record when connecting to MongoDB by specifying
+   the ``+srv`` modification in your connection string, you enable
+   TLS on your connection by default.
+
+Within your ``Config`` instance, you can set optional
+fields to configure TLS on your connection. For **testing purposes**,
+you can set the ``InsecureSkipVerify`` field to ``true``.
+
+Setting the ``InsecureSkipVerify`` field to ``true`` disables
+both certificate and hostname validation.
+   
+.. warning::
+
+   Specifying this option in a production environment makes
+   your application insecure and potentially
+   vulnerable to expired certificates and foreign processes posing
+   as valid client instances.
+
+For a full list of client options, see :ref:`golang-connection-options`.
+
+.. _golang-configure-tls-certificates:
+
+Configure Certificates
+----------------------
+
+To successfully initiate a TLS request, an application must prove its
+identity by referencing cryptographic certificates. To connect to
+MongoDB with TLS, your certificates must be stored as PEM
+files.
+
+.. important::
+
+   For production use, we recommend that your MongoDB deployment uses valid
```

Comment:
I: ["recommend" is hypothetical -> subjunctive mood](https://www.grammarly.com/blog/english-subjunctive-what-is-it/)

```suggestion
   For production use, we recommend that your MongoDB deployment use valid
```

Reply:
good catch!